scripts/encode_data.py

```diff
@@ -17,16 +17,21 @@
 
 class CustomDataLoader:
 
-    def __init__(self, chunk_size=500) -> None:
+    def __init__(self, chunk_size=500, parent_chunk_size=1000, child_chunk_size=100, do_parent_document=False) -> None:
```

Comment:
Can you add to the code comment explaining what the parent_doc does. I'd also prefer a name like use_parent_docs over do_parent_documents.